### PR TITLE
Prometheus image

### LINKED ISSUE #120: Create a Dockerfile for Prometheus
- Modification of yml config file should force the rebuild of the image in the pipeline
- Exactly like any other image in the pipeline

### PR DESCRIPTION
**Pull request by szymongalecki**
### Changes
- Prometheus has its own directory with Dockerfile and config file
- Configuration is burnt into image after building, there is no need to copy config file to droplet and mount onto container
- I tested that it works and temporarily put image on my docker hub

### CODE REVIEW

**Comment by dprzygocka** [comment]
i will rebuild the image, until i do it do not merge

**Review by dprzygocka — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -48,12 +48,12 @@ services:
       - postgres
 
   prometheus:
-    image: prom/prometheus
+    # built on apple sillicon, needs to be rebuild to run in droplet
+    image: szymongalecki/prometheus
```
Comment (dprzygocka) [inline docker-compose.yml:52]: image: petrastm/prometheus

File: docker-compose.yml [hunk 2]
```diff
@@ -48,12 +48,12 @@ services:
...
     container_name: prometheus
-    volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml
 
   grafana:
+    # correct architecture, runs in droplet, can be forked
     image: szymongalecki/grafana
```
Comment (dprzygocka) [inline docker-compose.yml:57]: image: petrastm/grafana

File: docker-compose.yml (hunk 1 above)
Comment (szymongalecki) [reply docker-compose.yml:52]: This is a local docker-compose, you probably want to change the one in remote-files directory

File: docker-compose.yml (hunk 2 above)
Comment (szymongalecki) [reply docker-compose.yml:57]: This is a local docker-compose, you probably want to change the one in remote-files directory

**Pull request merged**